.github/workflows/IR.yml

```
@@ -34,8 +34,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install ".[dev]"
-          solc-select install all
-          solc-select use 0.8.11
+          solc-select use 0.8.11--always-install
```

There should be a space between the version and the `--always-install` flag:

```suggestion
          solc-select use 0.8.11 --always-install
```

---

oh ok thankyou! sorry for the mess